### DISCUSSION TITLE
Welcome to PyBaMM Discussions!

### DISCUSSION THREAD

**Discussion by valentinsulzer** [pybamm-team/PyBaMM#1284] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about (like new battery models!).
  * Share ideas.
  * Engage with other community members.
  * [Welcome others and are open-minded](https://github.com/pybamm-team/PyBaMM/blob/develop/CODE-OF-CONDUCT.md). Remember that this is a community we build together 💪.

  For quicker (but less permanent) discussions, you can join our slack channel by emailing pybamm@gmail.com

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by valentinsulzer** [comment]
Hi, I'm Valentin (Tino) Sulzer, currently a postdoc at University of Michigan. 
I helped create PyBaMM back in 2018 while a PhD student at Oxford, and continue to actively develop and maintain it.

My main research focus is creating accurate and fast models for batteries and fuel cells, using reduced-order modeling (e.g. asymptotic analysis), efficient numerical methods, and (more recently) machine learning.

You can reach out to me by email at vsulzer@umich.edu, on Twitter [@tinosulzer](https://twitter.com/tinosulzer), or on the pybamm slack

**Comment by brosaplanella** [comment]
Hi everyone! I am Ferran, and I am a Research Fellow at the University of Warwick (UK) and I am part of the "Multi-Scale Modelling" project of The Faraday Institution. My work is on model reduction and parameterization, and I have been contributing to PyBaMM for almost a year now.

My work is on model reduction and parameterization, mainly for battery control applications. More generally, my research interests are on mathematical modelling with applications to industry, with a particular focus on sustainable technologies. I am also interested in working closely with experimentalists to bring together the outputs of the models and experiments, and help bridging the gap between both disciplines. For more information on what I do you can check my website [www.brosaplanella.com](https://www.brosaplanella.com).

Feel free to contact me at [Ferran.Brosa-Planella@warwick.ac.uk](mailto:Ferran.Brosa-Planella@warwick.ac.uk), on Twitter [@brosaplanella](https://twitter.com/brosaplanella) or on Slack.

**Comment by TomTranter** [comment]
Hi, I'm Tom Tranter, a post-doc in chemical engineering at UCL in the Electrochemical Innovation Lab (EIL) https://www.ucl.ac.uk/electrochemical-innovation-lab/ run by Professors Dan Brett and Paul Shearing. I work on modelling batteries and fuel cells and specialize in flow through porous media and also modelling thermal problems with up-scaled electrochemical models applied to cells and packs. I have also had previous roles in industry both as a programmer and a renewable energy specialist. I believe strongly in collaboration and have been very fortunate to be part of this wonderful open source battery modelling project since starting my research with the Faraday Institution https://faraday.ac.uk/

**Comment by rtimms** [comment]
Hi, I'm Robert Timms a postdoc at the [Mathematical Institute](https://www.maths.ox.ac.uk/), University of Oxford. I have been involved with the development and maintenance of PyBaMM since late 2018.

My current research focuses on the Mathematical Modelling of Batteries, and is being conducted within the Multi-Scale Modelling Fast Start Project of the [Faraday Institution](https://faraday.ac.uk/). More generally, I am interested in the use of asymptotic methods applied to practical problems arising from industry which couple mechanics, heat flow and chemistry. For more information you can visit my [website](https://www.robertwtimms.com/).

You can get in touch via email [timms@maths.ox.ac.uk](mailto:timms@maths.ox.ac.uk), twitter [@robertwtimms](https://twitter.com/robertwtimms), or on the PyBaMM slack.
